typed/union.go

```diff
@@ -243,7 +243,7 @@ func (u *union) Normalize(old, new, out value.Map) error {
 		return fmt.Errorf("multiple fields set without discriminator change: %v", ns)
 	}
 
-	// Update discriminiator if it needs to be deduced.
+	// Set discriminiator if it needs to be deduced.
```

Comment:
:-) Your script went crazy didn't it?

Reply:
Heh. Sigh. I think I did a editor based refactor when I saw it had been change to Set, but then when I merged it was no longer named Set :P